docs: stacker manual fast fix for style uniformity

jwwojak opened the pull request:
# Overview

One small front-matter change from a Slack convo + a few other minor adjustments to make manual similar to others.

## Test Plan and Hands on Testing

—

## Changelog

- For `index.md`, change header code to suppress the TOC and use module full title on first mention, "Stacker Module GEN1 Instruction Manual."
- Update all other pages to use short form title, "Stacker Module"
- Update front page code to match other manuals and remove comments
- Removed Flex logo image, go with title only on home page as discussed
- Update warranty page to match other module manuals

## Review requests

Please take a look. Small to trivial changes only.

## Risk assessment

Low, minor style adjustments only.

jwwojak committed:
Fast fix - small changes

- Update front matter
- Update front page and remove comments
- Update title header
- Update warranty page to match other modules
- Removed logo image

ecormany commented on docs/stacker-manual/docs/warranty.md:
```diff
@@ -20,21 +20,16 @@ Control your liquid handling robot and module using the Opentrons App. Download
 
-[**Opentrons Labworks Inc**](https://opentrons.com/)<br>
-45-18 Court Square West<br>
+Opentrons Labworks, Inc.<br>
```
I think there was no punctuation here on the theory that it's a mailing address and that's [what USPS prefers](https://pe.usps.com/text/pub28/28c2_007.htm). But that's a them thing, not an us thing. Punctuating at least our company name looks _right to us_.

jwwojak replied:
I will fix this.

Aside: went down the rabbit hole with punctuation and "Inc." in the _Chicago_ manual. Turns out, the comma is not required. § 6.44, "Commas are not required with _Inc._, _Ltd._, and such as part of a company's name."

They do indicate (§10.23 "Commonly used generic abbreviations for firms and companies") that a period is required after it, but Opentrons has transcended that limitation and moved on to a post-humanist future.

jwwojak committed:
Front-matter name change

Clarity around the formal name used in the frontmatter.

index.md
---
title: "Opentrons Flex Stacker Instruction Manual"
hide: "TOC"
---

subsequent pages

---
title: "Flex Stacker: section name here"
---

jwwojak committed:
Remove punctuation from company name

Don't need "..., Inc." Removing in favor of "Opentrons Labworks Inc"
